doc/guide/dynamics/dynamics-time.rst

```diff
@@ -211,62 +230,73 @@ Or equivalently:
+However, in many cases, we would like study the same problem with a range of parameters and
+not have to worry about manually changing the values on each run.
+QuTiP allows you to accomplish this using by adding extra arguments to coefficients
+function that make the :obj:`.QobjEvo`. For instance, instead of explicitly writing
+9 for the amplitude and 5 for the width of the gaussian driving term, we can add an
+`args` positional variable:
 
 
-.. plot::
-    :context: close-figs
+.. code-block:: python
 
-    def H1_coeff(t, args):
-        return args['A'] * np.exp(-(t/args['sigma'])**2)
+    >>> def H1_coeff(t, args):
+    >>>     return args['A'] * np.exp(-(t/args['sigma'])**2)
 
 
 or, new from v5, add the extra parameter directly:
 
 
-.. plot::
-    :context: close-figs
+.. code-block:: python
 
-    def H1_coeff(t, A, sigma):
-        return A * np.exp(-(t / sigma)**2)
+    >>> def H1_coeff(t, A, sigma):
+    >>>     return A * np.exp(-(t / sigma)**2)
 
 
-When the second positional input of the coefficient function is named ``args``, the arguments are passed as a Python dictionary of ``key: value`` pairs.
+When the second positional input of the coefficient function is named ``args``,
+the arguments are passed as a Python dictionary of ``key: value`` pairs.
 Otherwise the coefficient function is called as ``coeff(t, **args)``.
-In the last example, ``args = {'A': a, 'sigma': b}`` where ``a`` and ``b`` are the two parameters for the amplitude and width, respectively.
-This ``args`` dictionary need to be given at creation of the :obj:`.QobjEvo` when function using then are included:
+In the last example, ``args = {'A': a, 'sigma': b}`` where ``a`` and ``b`` are the
+two parameters for the amplitude and width, respectively.
+This ``args`` dictionary need to be given at creation of the :obj:`.QobjEvo` when
+function using then are included:
 
-.. plot::
-    :context: close-figs
+.. code-block:: python
 
-    system = [H0, [H1, H1_coeff]]
-    args={'A': 9, 'sigma': 5}
-    qevo = QobjEvo(system, args=args)
+    >>> system = [sigmaz(), [sigmax(), H1_coeff]]
+    >>> args={'A': 9, 'sigma': 5}
+    >>> qevo = QobjEvo(system, args=args)
 
 But without ``args``, the :obj:`.QobjEvo` creation will fail:
 
-.. plot::
-    :context: close-figs
-
-    try:
-        QobjEvo(system)
-    except TypeError as err:
-        print(err)
+.. code-block:: python
 
-When evaluation the :obj:`.QobjEvo` at a time, new arguments can be passed either with the ``args`` dictionary positional arguments, or with specific keywords arguments:
+    >>> QobjEvo(system)
+    TypeError: H1_coeff() missing 2 required positional arguments: 'A' and 'sigma'
 
-.. plot::
-    :context: close-figs
+When evaluation the :obj:`.QobjEvo` at a time, new arguments can be passed either
+with the ``args`` dictionary positional arguments, or with specific keywords arguments:
 
-    print(qevo(1))
-    print(qevo(1, {"A": 5, "sigma": 0.2}))
-    print(qevo(1, A=5))
+.. code-block:: python
```

Comment:
It would be nice to keep these as runnable code if we can so we know the examples keep working.

Reply:
The plot directive only output plot...
Here the print command were there, but only the print, not the output...

Reply:
Oh. I see. Maybe there is something we can add to the plot directive to make it output the console text too?

Reply:
I have not found any.
It would be great to have jupyter like cell in the documentation.

Reply:
Perhaps let's leave these as is for now and just make a TODO somewhere to find a solution.